.vscode/tasks.json

```diff
@@ -0,0 +1,10 @@
+{
+    "tasks": [
+        {
+            "label": "start debug listener",
+            "type": ,
```

Comment:
The empty type looks weird here. If we're running a shell command, shouldn't this type be `shell`?

Reply:
it was. I just removed that file. I wanted to have the debugger start with the start of the devcontainer. did not find any solution for that, so you always have to start the debugger first, before doing running your stuff or you will get these "could not connect to debugger" warnings everywhere